Project.toml

```
@@ -2,21 +2,22 @@ name = "HDF5"
 uuid = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 version = "0.16.14"
 
+[compat]
+Compat = "3.1.0, 4"
+HDF5_jll = "~1.10.5, ~1.12.0"
+Requires = "1.0"
+julia = "1.3"
```

```suggestion
Compat = "3.1.0, 4"
HDF5_jll = "~1.10.5, ~1.12.0"
Preferences = "1.3"
Requires = "1.0"
julia = "1.3"
```
Add compat for Preferences?